docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "Machine learning with the ACK SageMaker controller"
+description: "Train a machine learning model with the ACK service controller for Amazon SageMaker using Amazon Elastic Kubernetes Service"
+lead: ""
+draft: false
+menu: 
+  docs:
+    parent: "tutorials"
+weight: 40
+toc: true
+---
+
+The SageMaker ACK service controller makes it easier for machine learning developers and data scientists who use Kubernetes as their control plane to train, tune, and deploy machine learning models in Amazon SageMaker without logging into the SageMaker console. 
+
+The following steps will guide you through the setup and use of the Amazon SageMaker ACK service controller for training a machine learning model.
+
+## Setup
+
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) with ACK, this guide assumes that you have access to an Amazon EKS cluster. For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS - `eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with Amazon EC2 Linux managed nodes. If this is your first time creating an Amazon EKS cluster, see [Amazon EKS Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html).
```

Comment:
Some people on the team had trouble starting up their EKS clusters when we ran through this and found that the Linux managed nodes steps worked because the other eksctl steps didn't create nodes.

Reply:
Yes, the main creating section has command to [create without-nodegroup](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html). so people have always missed adding worker nodes. this is the only other simple set of commands we found to create a cluster with worker nodes